zerver/lib/actions.py

```diff
@@ -821,17 +821,18 @@ def do_send_typing_notification(notification):
     # type: (Dict[str, Any]) -> None
     recipient_user_profiles = get_recipient_user_profiles(notification['recipient'],
                                                           notification['sender'].id)
-    recipients = [{'id': profile.id, 'email': profile.email} for profile in recipient_user_profiles]
-    active_recipients = [profile for profile in recipient_user_profiles if profile.is_active]
-    sender = {'id': notification['sender'].id, 'email': notification['sender'].email}
+    active_recipient_ids = [profile.id for profile in recipient_user_profiles if profile.is_active]
+    sender_dict = {'id': notification['sender'].id, 'email': notification['sender'].email}
+    # Include a list of recipients in the event body to help identify where the typing is happening
+    recipient_dicts = [{'id': profile.id, 'email': profile.email} for profile in recipient_user_profiles]
```

Comment:
With some of @showell's PRs, we've been moving everything towards using `user_id` as the canonical name in these event dicts for a user ID (basically it's everywhere except for messages themselves); can you change this to use that style?  Otherwise looks great!